packages/serverless/src/backend_connector.js

```diff
@@ -316,6 +324,8 @@ function send(resourcePath, payload, finalLambdaRequest, callback) {
     // We already print the warning that a timeout happened
     // https://nodejs.org/api/http.html#requestdestroyed
     if (req.destroyed) {
+      // CASE: connection refused e.g. proxy or BE is not up, but we need to check if the cb was called
+      handleCallback();
```

Comment:
The problem was that we tried to send data to an endpoint which is offline and we received an errror, but we haven't called the customer's callback. I guess that http proxy agent 7.x release changed something internally and that's why the bug wasn't visible before the update.

Reply:
Yes, this makes sense. I think previously we only ever ran into the case that the request had already been destroyed when the callback also had already been called. But making sure it is getting called makes sense here, absolutely 👍